yarn.lock

```diff
@@ -2396,12 +2396,12 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@types/react-dom@npm:18.0.11":
-  version: 18.0.11
-  resolution: "@types/react-dom@npm:18.0.11"
+"@types/react-dom@npm:18.2.1":
```

Comment:
noise: were dependency updates required?

Reply:
There was a fix for that merged recently; if you didn't need to update any deps doing `git checkout origin/master -- yarn.lock` should clear everything up